plugins/topology/src/components/Topology/TopologySideBar/PLRlastUpdated.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import { fromNow } from '../../../common/utils/datetime';
+
+type PLRlastUpdatedProps = { plr: any };
+
+const PLRlastUpdated = ({ plr }: PLRlastUpdatedProps) => {
+  const {
+    metadata: { creationTimestamp },
+    status,
+  } = plr;
+  const lastUpdated = status
+    ? status.completionTime || status.startTime || creationTimestamp
+    : creationTimestamp;
```

Comment:
this doesn't seem right `creationTimestamp` is both cases?

Reply:
it looks like even when status is present completionTime, startTime both might not be available and that is why in that case also we are using creationTimestamp. And when status is not present we directly use creationTimestamp. This can be simplified though.